### PR TITLE
feat: Update home feed and add content

### PR DESCRIPTION
**Pull request by garypretty**
## Description
Update the feed content for the home page and add associated content.

#minor

### CODE REVIEW

**Review by cwhitten — COMMENTED** [review]
we'll need to find a way to localize this stuff.

File: feed.json [hunk 1]
```diff
@@ -1,87 +1,75 @@
...
-                    "title": "Using LG",
-                    "description": "Explore Language Generation and how to generate dynamic responses",
-                    "url": "https://www.youtube.com/watch?v=hiIiZnRcCv0"
-                }
-            ]
-        }
-    ]
+	"version": 1,
+	"whatsNewLinks": [{
+		"title": "Bot Framework Composer 1.3",
```
Comment (zhixzhan) [inline feed.json:4]: Latest composer version is 1.4, is this out of date ?

File: feed.json [hunk 2]
```diff
@@ -1,87 +1,75 @@
...
+		"viewAllLinkUrl": "https://www.youtube.com/channel/UC8qPRh20PpuxBvxf0w2WrwQ",
+		"cards": [{
+			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/composer-tutorials.png",
+			"title": "Using Prompts",
+			"description": "Walkthrough and tips for using Prompts to accept information from users.",
+			"url": "https://www.youtube.com/watch?v=dUT8L4_bVrk"
+		}, {
+			"image": "https://github.com/microsoft/BotFramework-Composer/raw/main/docs/home-feed-content/composer-tutorials.png",
+			"title": "LG for Advanced Responses",
+			"description": "How to use LG (Language Generation) to create dyanmic responses.",
```
[the remark this answers is not in the document]
Comment (zhixzhan) [reply feed.json:61]: For this images I guess it's better to put on some CDN, github.com maybe blocked in some areas.

[the remark this answers is not in the document]
Comment (zhixzhan) [reply feed.json:61]: For this images I guess it's better to put on some CDN, github.com maybe blocked in some areas.

**Comment by zhixzhan** [comment]
I have tested those content in latest working branch, all looks good.

File: feed.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (garypretty) [reply feed.json:61]: @zhixzhan we are not using a CDN for the feed at this point, given the complexity in setup. We will review this as needed, so we won't use one for the images either.  If the feed is unavailable, the application will continue to function.

[the remark this answers is not in the document]
Comment (garypretty) [reply feed.json:61]: Discussed with @cwhitten offline and confirmed we have deliberately not localized the blogs / articles / videos (feed content), however, other critical homepage content such as the calls to action for emulator / github will be localized as normal. We can review this decision in the future if needed.

File: feed.json (hunk 1 above)
Comment (garypretty) [reply feed.json:4]: We don't currently have a 1.4 blog post - it will be added to the feed once available.

**Review by a-b-r-o-w-n — APPROVED** [review]

**Pull request merged**